.github/workflows/terraform.yaml

```
@@ -34,7 +39,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 1
-          submodules: true
+          submodules: inputs.enable_submodules
```

let's use below syntax.
```suggestion
          submodules: ${{ inputs.enable_submodules }}
```